src/Compilers/CSharp/Test/Syntax/Parsing/UsingDirectiveParsingTests.cs

```diff
@@ -863,46 +863,6 @@ public void UsingUnsafeNonAlias()
         EOF();
     }
 
-    [Fact]
```

Comment:
this test moved down so it could remain with the rest of the variants added for it.  I couldn't add teh variants next to it, or github blewup thinking all the tests had changed.